css/main.css

```diff
@@ -4,11 +4,19 @@
    Author's custom styles
    ========================================================================== */
 
-
-
-
-
-
+/**** Global styles for Acessibilty ****/
+
+body {
+   color: black;
+	font-family: arial;
```

Comment:
Consider adding `font family: Arial, "Helvetica Neue", Helvetica, sans-serif;` to cover all browser font availability and rendering issues.